mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -966,4 +966,41 @@ private String getCurrentSnapshotForHiveCatalogTable(org.apache.iceberg.Table ic
     return ((BaseMetastoreTableOperations) ((BaseTable) icebergTable).operations())
         .currentMetadataLocation();
   }
+
+  @Test
+  public void testCreateTableWithPercentInName() throws IOException {
+    Assume.assumeTrue("This test requires Hive Version 4.", HiveVersion.min(HiveVersion.HIVE_4));
```

Comment:
is this test even applicable then for the Iceberg codebase? Iceberg + Hive4 are maintained by the Hive community in https://github.com/apache/hive/tree/master/iceberg

Reply:
Probably not.  Someone from the Hive community asked me to contribute this to Iceberg as well.  Maybe I should just remove the test case.